src/App.css

```
@@ -1,9 +1,17 @@
 .app {
-  background: var(--bg-body);
+  /* background: var(--bg-body); */
+  background: radial-gradient(hsl(198, 69%, 23%), var(--bg-body), #175547);
+  background-size: 300% 300%;
+  animation: gradient 15s ease infinite;
   min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
```

Disable background animation when the user specified reduced motion on their OS settings.

```suggestion

@media screen and (prefers-reduced-motion) {
  .app {
    animation: none;
  }
}

```